.github/workflows/c-cpp_tmpl.yaml

```diff
@@ -6,19 +6,22 @@ on:
       dependencies:
         required: true
         type: string
+      clang-build:
+        required: true
+        type: boolean
```

Comment:
```suggestion
      clang-build:
        required: true
        type: boolean
      gcc-build:
        required: true
        type: boolean
```
I think it's better to have one variable per compiler, than trying to use `!clang-build` to mean GCC

Reply:
Ok, I can do that.